blog/settings.py

```diff
@@ -2,7 +2,7 @@
 
 import environ
 
-env = environ.Env(DEBUG=(bool, False),)
+env = environ.Env(DEBUG=(bool, True),)
```

Comment:
No, you have to react to comments instantly, otherwise you risk forgetting about them.